.circleci/config.yml

```diff
@@ -29,18 +29,31 @@ aliases:
     command: |
       RELEASE_TIMESTAMP="$(date +'%Y%m%d%H%M%S')"
       VPKG=$($(npm bin)/json -f package.json version)
-      export RELEASE_VERSION=${VPKG}-prerelease.${RELEASE_TIMESTAMP}
-      echo $RELEASE_VERSION
-      export NPM_TAG=latest
+      echo export RELEASE_VERSION=${VPKG}-prerelease.${RELEASE_TIMESTAMP} >> $BASH_ENV
+      echo export NPM_TAG=latest >> $BASH_ENV
       if [[ "$CIRCLE_BRANCH" == hotfix/* ]]; then # double brackets are important for matching the wildcard
-        export NPM_TAG=hotfix
+        echo export NPM_TAG=hotfix >> $BASH_ENV
       fi
   - &deploy-gh-pages
     name: "deploy to gh pages"
     command: |
       git config --global user.email $(git log --pretty=format:"%ae" -n1)
       git config --global user.name $(git log --pretty=format:"%an" -n1)
       npm run deploy -- -e $CIRCLE_BRANCH
+  - &deploy-npm
+    name: "deploy to npm"
+    command: |
+      echo "npm tag: $NPM_TAG"
+      echo "release version: $RELEASE_VERSION"
+      npm version --no-git-tag-version $RELEASE_VERSION
+      npm set //registry.npmjs.org/:_authToken=$NPM_TOKEN
+      npm publish --tag $NPM_TAG
+  - &tag-commit
+    name: "tag commit in github"
+    command: |
+      echo $RELEASE_VERSION
+      git tag $RELEASE_VERSION
+      git push $CIRCLE_REPOSITORY_URL $RELEASE_VERSION
```

Comment:
It's a minor point, but I feel like we don't need to fight `npm` with `--no-git-tag-version` here. I think a roughly-equivalent set of steps would be to flip the order that `tag-commit` and `deploy-npm` steps run in, and adjust them to look something like this:

```suggestion
      echo "release version: $RELEASE_VERSION"
      npm set //registry.npmjs.org/:_authToken=$NPM_TOKEN
      npm publish --tag $NPM_TAG
  - &tag-commit
    name: "tag commit in github"
    command: |
      echo $RELEASE_VERSION
      npm version $RELEASE_VERSION
      git push $CIRCLE_REPOSITORY_URL $RELEASE_VERSION
```

(I didn't flip the order here because I wanted the diff to be easier to read.)

In that case I think it would also make sense to rename `tag-commit` to something like `bump-version`.